.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "db"]
 	path = db
   url = https://github.com/unepwcmc/protectedplanet-db.git
-  branch = master
+  branch = merge-pame-database
```

Comment:
Can you make sure to change this to master again once everything is merge into master in the pp-db repo please?
Happy to merge after that.

Reply:
Sure thing! I will change it back to `master` once pp-db is merged into master!